### PR TITLE
ResStock-HPXML: Occupancy adjustments

### PR DESCRIPTION
**Pull request by shorowit**
Resolves #[issue number here].

## Pull Request Description

- [ ] **Appliances** - Incorporating # occupants adjustment factors on top of the base ANSI 301 equations. Should provide better agreement (but will never give perfect agreement).
- [ ] **MELs/MGLs/LULs** - The BAHSP-based equations are currently preserved in ResStock-HPXML, yielding perfect agreement. But should we switch to # occupants adjustment factors on top of the ANSI 301 equations as we're doing for appliances?
- [x] ~~**Lighting** - Equations have never been based on # occupants. ANSI 301 equations were [backported to current ResStock](https://github.com/NREL/resstock/pull/619), yielding perfect agreement.~~

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Comment by shorowit** [comment]
This had an excellent improvement on hot water usage.

## Before

image
image

## After

image
image

Unfortunately appliance energy use was not similarly improved. Need to dig further into that.

**Comment by shorowit** [comment]
Results before and after this change:

### Clothes dryer

image

### Clothes washer

image

### Dishwasher

image

### Cooking range/oven

image

**Comment by shorowit** [comment]
TODO
- [ ] Show comparison of BA to 301 plug load equations
- [ ] Double-check equivalency of ResStock vs ResStock-HPXML appliance options (especially for new/removed arguments)

**Comment by ejhw** [comment]
Talked to @afontani . We are good with transitioning to use 301 equations for resstock going forward.

**Comment by shorowit** [comment]
@ejhw Here's a comparison of ResStock/RECS vs BAHSP vs ANSI 301:

image

Spreadsheet: [PlugLoadsComparison.xlsx](https://github.com/NREL/resstock/files/6844343/PlugLoadsComparison.xlsx)

Note that it's not a perfect comparison because, for ResStock, it uses a [calculated N_occupants based on N_bedrooms](https://github.com/NREL/resstock/blob/742ab5837ed477501ef94832ac2182edec7ca056/resources/measures/HPXMLtoOpenStudio/resources/geometry.rb#L1532-L1542), whereas these days ResStock actually [samples N_occupants based on N_bedrooms](https://github.com/NREL/resstock/blob/develop/project_national/housing_characteristics/Occupants.tsv).

Long story short, the ANSI 301 equations are quite a bit different for smaller/larger homes, so I suggest not changing ResStock. We should consider proposing that 301 adopt updated equations based on RECS.

**Comment by shorowit** [comment]
Merging this into restructure since this is an obvious improvement. We can continue to investigate differences going forward.
